app/public/index.html

```diff
@@ -19,7 +19,8 @@
     <script src="<%= BASE_URL %>js/gapi.js" crossorigin="anonymous" integrity="sha384-m0UrUl+xP4P/jooB7I71mIMQv8pSXopfvry4S8m1KJZTKCLv5wVy1h9JIADN2v8T"></script>
     <meta name="google-signin-client_id" content="876733105116-i0hj3s53qiio5k95prpfmj0hp0gmgtor.apps.googleusercontent.com" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" />
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons" />    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css" />
```

Comment:
please download the icons used and put them in the public folder

Reply:
Done

Reply:
Also, delete any unused icons in public/images folder (before merging.. not right now).
Don't delete any icons present in logos folder